AzureTable/Microsoft.DataTransfer.AzureTable/Sink/TableAPIBulkSinkAdapter.cs

```diff
@@ -58,11 +64,12 @@ public int MaxDegreeOfParallelism
 
             CloudTableClient tableClient = storageAccount.CreateCloudTableClient(connectionPolicy: connectionPolicy);
             cloudtable = tableClient.GetTableReference(_tableName);
-
             requestOptions = new TableRequestOptions()
             {
                 RetryPolicy = new ExponentialRetry(TimeSpan.FromSeconds(3), 3)
             };
+
+            remoteLogger = remoteLoggingClientProvider.CreateRemoteLoggingClient(storageAccount, connectionPolicy);
```

Comment:
Do you need to create this, even if remoteLogging argument is false? Would it be more practical to just keep this null and check that, instead of having a separate boolean field?

Reply:
N/A as this logic moved to core. If user specifices remoteLogging , that's when this will be initialized now.